pkg/metrics/server.go

```diff
@@ -14,13 +14,13 @@ import (
 
 	grpc_prometheus "github.com/grpc-ecosystem/go-grpc-prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/wlynch/slogctx"
```

Comment:
same comment as https://github.com/chainguard-dev/mono/pull/15471#issuecomment-1858364516

would be nice to move this under `chainguard-dev`